.github/workflows/node-js-ci.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        node-version: [12.x]
+        node-version: [12.x, 13.x, 14.x]
```

Comment:
I think only the even numbered releases are what people should be using for LTS.  I'd suggest 12, 14.

cc @manekenpix

Reply:
I agree with @humphd 
Take a look at [this](https://nodejs.org/en/about/releases/)

![image](https://user-images.githubusercontent.com/23108901/105233571-07145000-5b38-11eb-9880-60a519c5a4d1.png)